### PR TITLE
Change QuantizedFacings:s to 0

### LINKED ISSUE #20: Crash (Actor type 'zep' does not define a quantized body orientation)
``` csharp
Red Alert 2 Mod at Version {DEV_VERSION}
on map 8ad775a6b69b16351ff478f464c90dcca1e075fc (Breakaway by Westwood Studios).
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.34209
Exception of type `System.InvalidOperationException`: Actor type 'zep' does not define a quantized body orientation.
   at OpenRA.Mods.Common.Traits.BodyOrientation.<>c__DisplayClass3.<.ctor>b__0()
   at System.Lazy`1.CreateValue()
   at System.Lazy`1.LazyInitValue()
   at System.Lazy`1.get_Value()
   at OpenRA.Mods.Common.Traits.BodyOrientation.QuantizeOrientation(Actor self, WRot orientation)
   at OpenRA.Mods.Common.Traits.WithVoxelBody.<>c__DisplayClass9.<.ctor>b__1()
   at OpenRA.Graphics.VoxelRenderer.RenderAsync(WorldRenderer wr, IEnumerable`1 voxels, WRot camera, Single scale, Single[] groundNormal, WRot lightSource, Single[] lightAmbientColor, Single[] lightDiffuseColor, PaletteReference color, PaletteReference normals, PaletteReference shadowPalette) in d:\git\OpenRA-ra2-master\OpenRA.Game\Graphics\VoxelRenderer.cs:Line 111
   at OpenRA.Mods.Common.Graphics.VoxelRenderable.FinalizedVoxelRenderable..ctor(WorldRenderer wr, VoxelRenderable voxel)
   at OpenRA.Mods.Common.Graphics.VoxelRenderable.PrepareRender(WorldRenderer wr)
   at OpenRA.Graphics.WorldRenderer.<GenerateRenderables>b__6(IRenderable r) in d:\git\OpenRA-ra2-master\OpenRA.Game\Graphics\WorldRenderer.cs: 122
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at OpenRA.Graphics.WorldRenderer.GenerateRenderables() in d:\git\OpenRA-ra2-master\OpenRA.Game\Graphics\WorldRenderer.cs:Line 121
   at OpenRA.Graphics.WorldRenderer.Draw() in d:\git\OpenRA-ra2-master\OpenRA.Game\Graphics\WorldRenderer.cs:Line 138
   at OpenRA.Game.RenderTick() in d:\git\OpenRA-ra2-master\OpenRA.Game\Game.cs:Line 515
   at OpenRA.Game.Loop() in d:\git\OpenRA-ra2-master\OpenRA.Game\Game.cs:Line 643
   at OpenRA.Game.Run() in d:\git\OpenRA-ra2-master\OpenRA.Game\Game.cs:Line 661
   at OpenRA.Program.Run(String[] args) in d:\git\OpenRA-ra2-master\OpenRA.Game\Support\Program.cs:Line 117
   at OpenRA.Program.Main(String[] args) in d:\git\OpenRA-ra2-master\OpenRA.Game\Support\Program.cs:Line 39
```

### PR DESCRIPTION
**Pull request by MustaphaTR**
Fixes #20.

### CODE REVIEW

File: mods/ra2/rules/defaults.yaml [hunk 1]
```diff
@@ -392,6 +392,7 @@
 		Voice: Attack
 	ActorLostNotification:
 	BodyOrientation:
+		QuantizedFacings: 32
```
Comment (GraionDilach) [inline mods/ra2/rules/defaults.yaml]: TS uses `QuantizedFacings: 0` on voxels (it defaults to -1).

Comment (MustaphaTR) [inline mods/ra2/rules/defaults.yaml]: With voxels are you talking about all voxels or voxel aircraft, becuase i copied it from `^Vehicles` at here.

Comment (GraionDilach) [inline mods/ra2/rules/defaults.yaml]: https://github.com/OpenRA/OpenRA/blob/bleed/mods/ts/rules/defaults.yaml#L439

Comment (MustaphaTR) [inline mods/ra2/rules/defaults.yaml]: As i said, Voxels at here using `32` and it works with no problem but i'll test `0` too.

Edit: I tested but couldn't see any difference.

Comment (penev92) [inline mods/ra2/rules/defaults.yaml]: I think the point is there is no reason to limit it to 32 when it's actually a voxel and doesn't care about the number of facings. 32 is perfectly fine for sprite actors to look good, so limiting a voxel actor to 32 will leave it OK-looking, but there is still no reason to limit it.

Comment (MustaphaTR) [inline mods/ra2/rules/defaults.yaml]: OK, changing it then, but should i change ones on `^Vehicle` and `^Ship` on this PR or open a new PR for them.

Comment (penev92) [inline mods/ra2/rules/defaults.yaml]: Seems fine to do them in this PR and rename it.

**Comment by phrohdoh** [comment]
Cool, thanks!

**Pull request merged**